rest-model/src/main/java/org/jboss/pnc/rest/restmodel/bpm/BpmBuildConfigurationCreationRest.java

```diff
@@ -91,4 +94,8 @@
     @Getter
     @Setter
     private Set<Integer> buildConfigurationSetIds;
+    
+    @Getter
+    @Setter
+    private Map<String, String> genericParameters ;
```

Comment:
could you remove the space between `s` and `;`? (btw in some next change you'll make).